migrations/20170508160723_app_workload.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('workload', function (table) {
+    table.increments('id')
+    table.bigInteger('workload_owner_id').references('id').inTable('workload_owner')
+    table.bigInteger('workload_report_id').references('id').inTable('workload_report')

```

Comment:
Will need to include all of the aggregate columns that are in the current WMT workload table, that we haven't moved into any of the Comm/Lic/CusTier tables.

I.e. things like TotalPoints and so on.

I'll add this list to the Schema Changes doc, as this information is currently only in the existing table definitions.